src/olympia/amo/urlresolvers.py

```diff
@@ -36,8 +36,15 @@ def split_path(path_):
         first_lower = first.lower()
         lang, dash, territory = first_lower.partition('-')
 
-        # Check language-territory first.
-        if first_lower in settings.LANGUAGE_URL_MAP:
+        # First test shorter languages shortcuts.
+        if not dash and first in settings.SHORTER_LANGUAGES:
+            first = settings.SHORTER_LANGUAGES[first]
+            if second in amo.APPS:
+                return first, second, rest
+            else:
+                return first, '', first_rest
+        # Then check language-territory.
+        elif first_lower in settings.LANGUAGE_URL_MAP:
```

Comment:
Nit: the elif could be returned to an if since the first if definitely returns right? I tend to prefer split if statements if they all return since elif makes me think we could continue executing after the final branch, but in this case there is a (not totally obvious) return

Reply:
but I guess there are already other if/elif/elif branches so nvm 🤷